tests/test_gateway.py

```diff
@@ -145,6 +146,10 @@ def init_gateway(monkeypatch):
     GatewayClient.clear_instance()
 
 
+@pytest.mark.skipif(
+    sys.version_info < (3, 6),
+    reason="Kernel manager is AsyncMappingKernelManager, Python version < 3.6"
+)
```

Comment:
Per the previous comment regarding `GatewayKernelManager`, these tests should be opened up on 3.5.  Same goes for the remaining tests in this module.

Reply:
Done.